src/components/LoanCards/KivaClassicBasicLoanCardExp.vue

```diff
@@ -276,8 +276,8 @@ export default {
 			}
 
 			this.isLoading = false;
-			this.loan = result.data?.lend?.loan || null;
-			this.basketItems = result.data?.shop?.basket?.items?.values || null;
+			this.loan = result.data?.lend?.loan ?? {};
+			this.basketItems = result.data?.shop?.basket?.items?.values ?? [];
```

Comment:
Using nullish operators ensure there is at least an object or array present while the loans are loading.